boards/arm/cyclonev_socdk/cyclonev_socdk_defconfig

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Description:
+# Cyclone V SoC Development Kit Board configuration file
+
+CONFIG_SOC_SERIES_CYCLONE5=y
+CONFIG_SOC_CYCLONE5=y
+CONFIG_BOARD_CVSXDEVKIT=y
+CONFIG_ARM_ARCH_TIMER=y
+
+CONFIG_SYSCON=y
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_NS16550=y
+CONFIG_UART_NS16750_EXTFIFO=y
+#CONFIG_UART_INTERRUPT_DRIVEN=y
+#CONFIG_UART_NS16550_ACCESS_WORD_ONLY=y
+#CONFIG_UART_ALTERA_JTAG=y
+
+#Config Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_PRINTK=y
+
+#Flash not used.
+CONFIG_FLASH_SIZE=0
+CONFIG_FLASH_BASE_ADDRESS=0x0
+
+# GPIOs
+CONFIG_GPIO=y
+CONFIG_GPIO_DW=y
+CONFIG_GPIO_DW_0=y
+CONFIG_GPIO_DW_1=y
+CONFIG_GPIO_DW_2=y
+
+# I2C:
+CONFIG_I2C=y
+CONFIG_I2C_DW=y
```

Comment:
this can be added in the Kconfig.defconfig file

if I2C
config I2C_DW
	default y
endif /* I2C */


and for all others below...

Reply:
This is solved